src/utils/signaling.js

```diff
@@ -1041,12 +1041,12 @@ Signaling.Standalone.prototype.helloResponseReceived = function(data) {
 
 	if (!this.settings.helloAuthParams.internal && (!this.hasFeature('audio-video-permissions') || !this.hasFeature('incall-all') || !this.hasFeature('switchto'))) {
 		showError(
-			t('spreed', 'The configured signaling server needs to be updated to be compatible with this version of Talk. Please contact your administrator.'),
```

Comment:
Why is that? Is there a rule for it? I think it's quite unlucky to remove a dot as it breaks all languages. Also it's a full sentence, so it should have a dot, no?

Reply:
With error messages consisting of two or more sentences, there is always the problem of whether there should be a dot at the end of the sentence.
In my opinion, multi-sentence error messages should be avoided. The message should be worded in one sentence.
Some applications use a dot at the end of the next sentence, while some do not. It is important that the spelling rules are the same for all messages.
The change has been undone.

Reply:
I guess https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html

Then again, why would a full sentence not end with a dot? Only when it's in a toast?
Is it different when it's shown in a modal, or any other UI element? That's such a weird rule and it will be broken all the time (especially by new apps and when community sends PRs, etc) and until it's then noticed and fixed (like here) community translated it already, but then it's being broken by removing the trailing dot and shows up untranslated again unless translators translate it a second time...

Reply:
My question remains, when should the dot at the end of a full sentence be skipped and why?